### ISSUE TITLE
Ubuntu WSL: write alt text for images

### ISSUE THREAD

**Issue by edibotopic** [canonical/open-documentation-academy#119]
This issue requires writing appropriate alt text for images in the Ubuntu WSL docs.
Most of the images contain no alt text, so you will need to write new alt text from scratch.

[Ubuntu WSL docs](https://canonical-ubuntu-wsl.readthedocs-hosted.com/en/latest/)

# Background

Several Tutorials and How-to guides in the Ubuntu WSL documentation contain images. Many of these images do not have appropriate alt text. This is an accessibility issue that could make the documentation difficult to interpret for people using screen reader technology.

A file has been prepared in the Canonical Open Documentation Academy repo that contains these images and some additional background. You will edit this file to include alt text for the images.

# Prerequisites

Alt text is concise, so this issue will not involve a large volume of writing; however, the alt text needs to be written carefully to reflect the contents of a given image. While the content of some images will be self-evident, others will require you to refer to the Ubuntu WSL documentation.

Access to a Windows machine and a familiarity with WSL would be helpful for understanding the content and testing any commands. Note, however, that the specific issues outlined above can be addressed without significant experience of WSL or access to a Windows machine.

**Comment by davidekete** [comment]
Hi @edibotopic, I'd like to work on this.

**Comment by davidekete** [comment]
Hi @edibotopic, I should be done before Monday from the looks of the file. Is that okay?
Also, to be clear, I'm editing the file in the description to include the alt text, not the actual documentation. Is that correct?

**Comment by edibotopic** [comment]
Hi again @davidekete.

Monday would be OK but don't put yourself under too much pressure.
If you find yourself having difficulty with particular images and need to dive into the documentation a little deeper it might end up taking more time. Just keep me updated and that will be fine.

Yes, you are right. I spent some time combining the images into that [this file](https://github.com/canonical/open-documentation-academy/blob/main/wsl/accessibility/ubuntu-wsl-doc-images.md) so that you could edit the alt text directly in a single location.

When you are happy with your proposed changes you can make a PR directly to this repo (Canonical Open Documentation Academy) and I will review. Later, I will then merge your changes to the Ubuntu WSL docs, credit you in the PR and link it here.

Let me know if you have any other questions.

**Comment by davidekete** [comment]
Hi @edibotopic, just keeping you in the loop like you asked. I'm working on it but haven't finished it as I predicted. However, I should be able to turn it in soon.

**Comment by edibotopic** [comment]
Thanks for the update @davidekete . No worries: it's careful work!

I likely won't be able to review properly until next week so please take your time.

**Comment by davidekete** [comment]
> Thanks for the update @davidekete . No worries: it's careful work!
> 
> I likely won't be able to review properly until next week so please take your time.

No problem.

**Comment by davidekete** [comment]
Hi @edibotopic, here's the link to the PR:  https://github.com/canonical/open-documentation-academy/pull/124

**Comment by edibotopic** [comment]
@davidekete -- I'm back from holidays today and will begin looking at this. Thank you.

**Comment by edibotopic** [comment]
The contribution of @davidekete has been [merged into the official Ubuntu WSL documentation](https://github.com/ubuntu/WSL/pull/490).

Thanks again David! Nice working with you.
